go.mod

```diff
@@ -5,76 +5,126 @@ go 1.17
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo/v2 v2.1.3
-	github.com/onsi/gomega v1.17.0
+	github.com/onsi/gomega v1.19.0
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/operator-framework/api v0.10.0
+	github.com/operator-framework/api v0.16.0
```

Comment:
updation of operator framework api should be in a different PR

Reply:
We are not updating that package as it is mentioned in replaces section
https://github.com/red-hat-storage/managed-fusion-agent/pull/18/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R124